icons/badge-minus.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
+  <line x1="8" x2="16" y1="12" y2="12" />
```

Comment:
Maybe a smaller minus symbol fits better
```suggestion
  <path d="M9 12h6" />
```
![file-question (2)](https://user-images.githubusercontent.com/25524993/233154760-57611504-e1e5-47a7-b2cd-88e5e9dfbb9e.svg)

Reply:
I think it should be the exact same as in all other similar icons (circle, square etc.) as long as the guidelines don't come into play.